### PR TITLE
defaults: fail if can not write ansible logs

### PR DESCRIPTION
**Pull request by leseb**
Signed-off-by: Sébastien Han <seb@redhat.com>

### CODE REVIEW

**Comment by leseb** [comment]
The entire fails because the patch works :p, we now need to decide if we make this optional or not...

**Comment by guits** [comment]
why would we want to make this test mandatory?

**Comment by leseb** [comment]
@guits this still needs to be discussed but we have been having many users reporting issues and not able to provide enough logs. So this might be useful. This is still DNM for me as we need to discuss this.

File: roles/ceph-defaults/tasks/ansible_logs.yml [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+  changed_when: false
+  failed_when: false
+  register: mktemp
+  when: ansible_cfg.stat.exists == True
+
+- name: fail if log_path directory from ansible.cfg is not writable
+  fail:
+    msg: "make sure log_path directory from ansible.cfg is writable by the user running ansible"
+  when:
+    - mktemp.stdout != '0'
```
Comment (guits) [inline roles/ceph-defaults/tasks/ansible_logs.yml]: shouldn't it be `mktemp.rc != 0` ?

File: roles/ceph-defaults/tasks/ansible_logs.yml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+    - ansible_cfg.stat.exists == True
```
Comment (guits) [inline roles/ceph-defaults/tasks/ansible_logs.yml]: shouldn't we stat for `{{ ansible_log_location }}` and check for `stat_ansible_log_location.stat.exists == False` instead ?

**Comment by guits** [comment]
jenkins test pipeline

**Comment by guits** [comment]
jenkins test pipeline

**Comment by guits** [comment]
jenkins test this please

**Pull request closed without merging**